static-site/src/components/ListResources/index.tsx

```diff
@@ -109,6 +123,7 @@ interface ListResourcesResponsiveProps {
   defaultGroupLinks: boolean
   groupDisplayNames: GroupDisplayNames
   showcase: Card[]
+  parserCallBackFxn: (response: Response) => Promise<{ pathPrefix: string; pathVersions: PathVersions; }>;
```

Comment:
The object type `{pathPrefix: string, pathVersions: PathVersions}` is used in many places, I think enough to warrant some standardization.

Suggestion: Add an interface in `types.ts` and use this in place of the object type.

```ts
interface ResourceListingInfo {
  pathPrefix: string
  pathVersions: PathVersions
}
```